docs/index.md

```diff
@@ -0,0 +1,3 @@
+# code-client
+
+Typescript consumer of public API
```

Comment:
Let's write of Snyk Code public api. I can imagine engineers browsing Backstage and not making sense of what public API is referenced here 😅